pkg/activator/net/revision_backends.go

```diff
@@ -272,7 +285,7 @@ func (rw *revisionWatcher) checkDests(curDests, prevDests dests) {
 		// precise load balancing in the throttler.
 		hs, noop, err := rw.probePodIPs(curDests.ready.Union(curDests.notReady))
 		if err != nil {
-			rw.logger.With(zap.Error(err)).Warnf("Failed probing: %+v", curDests)
+			rw.logger.Warnw("Failed probing pods", zap.Object("curDests", curDests), zap.Error(err))
```

Comment:
Same here, wouldn't `dests` be simpler, like below.

Reply:
We have cur and prev dests, so he'd I'd keep the prefix.